src/components/network/network-table.js

```diff
@@ -430,6 +467,8 @@ const NetworkTable = (props) => {
             filter,
             formatCell,
             props.disabled,
+            isLineOnEditMode,
```

Comment:
Slight confusion between these 2 terms : `isLineOnEditMode` and `isModifyingRow`

Reply:
Fixed.